reflex/state.py

```diff
@@ -352,17 +350,88 @@ def __init_subclass__(cls, **kwargs):
         events = {
             name: fn
             for name, fn in cls.__dict__.items()
-            if not name.startswith("_")
-            and isinstance(fn, Callable)
-            and not isinstance(fn, EventHandler)
+            if cls._item_is_event_handler(name, fn)
         }
+
+        for mixin in cls._mixins():
+            for name, value in mixin.__dict__.items():
+                if isinstance(value, ComputedVar):
+                    fget = cls._copy_fn(value.fget)
+                    newcv = ComputedVar(fget=fget, _var_name=value._var_name)
+                    newcv._var_set_state(cls)
+                    setattr(cls, name, newcv)
+                    cls.computed_vars[newcv._var_name] = newcv
+                    cls.vars[newcv._var_name] = newcv
+                    continue
+                if events.get(name) is not None:
+                    continue
+                if not cls._item_is_event_handler(name, value):
+                    continue
+                if parent_state is not None and parent_state.event_handlers.get(name):
+                    continue
+                value = cls._copy_fn(value)
```

Comment:
what is the value is not a function?

Reply:
Should be handled by the `_item_is_event_handler` condition in line 368